Tools/i18n/msgfmt.py

```diff
@@ -6,9 +6,9 @@
 This program converts a textual Uniforum-style message catalog (.po file) into
 a binary GNU catalog (.mo file).  This is essentially the same function as the
 GNU msgfmt program, however, it is a simpler implementation.  Currently it
-does not handle plural forms but it does handle message contexts.
+handles plural forms and message contexts, but does not generate hash table.
 
-Usage: msgfmt.py [OPTIONS] filename.po
+Usage: msgfmt.py [OPTIONS] filename.po [filename.po ...]
```

Comment:
This seems to be more common:
```suggestion
Usage: msgfmt.py [OPTIONS] filename.po ...
```